unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService/WebInterfaceEmotesCatalogService.cs

```diff
@@ -287,6 +293,10 @@ public async UniTask<EmbeddedEmotesSO> GetEmbeddedEmotes()
             return embeddedEmotesSO;
         }
 
+        public UniTask<IReadOnlyList<WearableItem>> RequestEmoteCollectionInBuilderAsync(IEnumerable<string> collectionIds,
+            CancellationToken cancellationToken, List<WearableItem> emoteBuffer = null) =>
+            throw new NotImplementedException("Implemented in LambdasEmotesCatalogService");
```

Comment:
and here (same concern of usage of this exception)

Reply:
`WebInterfaceEmotesCatalogService` and `WebInterfaceWearablesCatalogService` are marked as obsolete and could be removed after this implementation. They are only being used to preview wearables & emotes in the backpack v1. This implementation is deprecating that flow. So i think its over-engineering at this point to segregate into new interfaces.

Reply:
then it make sense 🟢 maybe bring also `TODO`